handbook/digital-experience/README.md

```diff
@@ -284,6 +284,7 @@ Follow these steps to archive any document:
 From time to time, you will need to schedule an interview between a candidate and the CEO:
 1. [Make a copy of the "¶¶ CEO interview template"](https://docs.google.com/document/d/1yARlH6iZY-cP9cQbmL3z6TbMy-Ii7lO64RbuolpWQzI/copy) (private Google doc)
 2. Change file name and heading of doc to `¶¶ CANDIDATE_NAME (CANDIDATE_TITLE) <> Mike McNeil, CEO final interview (YYYY-MM-DD)`
+   - Click the share button and share access with the CEO.
```

Comment:
Let's add a numbered step here instead of a sub-bullet. This is important enough and has been missed enough that it's worth it. Let's move this step down and have it happen right after you crate the copy

Reply:
With moving the doc to the "Reference checks and hiring data" we won't need this to be a step right? Because once it's placed there, Mike will have access.